### PR TITLE
PT-2234: Replace the full pedigree with its thumbnail in the Family history and pedigree section.

### PR DESCRIPTION
**Pull request by veronikaslc**
Fixed.

### CODE REVIEW

**Comment by veronikaslc** [comment]
Also fixes issues [PT-2235](https://phenotips.atlassian.net/browse/PT-2235) and [PT-2096](https://phenotips.atlassian.net/browse/PT-2096)

**Comment by veronikaslc** [comment]
@antonkatz , @allasm please review

File: components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml [hunk 1]
```diff
@@ -242,34 +373,34 @@
...
 #set ($pedigreeObj = $doc.getObject('PhenoTips.PedigreeClass'))
 #if ($pedigreeObj)
-  #set ($data = $!pedigreeObj.getProperty('data').value)
   #set ($image = $pedigreeObj.getProperty('image').value)
 #end## pedigree obj exists
-#if ($xcontext.action == 'edit' || "$!data" != '')
-  $xwiki.jsx.use('PhenoTips.UIX_Field__pedigree')##
 
-  === $services.localization.render('phenotips.patientRecord.pedigree.title') ===
+$xwiki.jsx.use('PhenoTips.UIX_Field__pedigree')##
```
Comment (antonkatz) [inline components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml]: Don't forget about this

File: components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml [hunk 2]
```diff
@@ -126,32 +136,17 @@
       pedigreeTrigger.observe("click", function(event) {
         event.stop();
         document.observe("xwiki:document:saved", function () {
-          window.self.location = pedigreeTrigger.href;
```
Comment (antonkatz) [inline components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml]: is ```pedigreeTrigger.href``` used anywhere else?

**Comment by antonkatz** [comment]
+1

File: components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml (hunk 2 above)
Comment (veronikaslc) [inline components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml]: nope

Comment (antonkatz) [inline components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml]: I can't remember why I asked. On a different note, why did you chose to hardcode ```location.search``` to ```?sheet=...```

Comment (veronikaslc) [inline components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml]: cause it works :) I am actually wondering if there is any better way to redirect to pedigree editor. @allasm ?

Comment (allasm) [inline components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml]: I'm not sure why you need .search, just .location should be enough I think?

Other than that, there must be an XWiki way to get the URL for pedigree editor without hardcoding, but I don't know it

**Comment by allasm** [comment]
+1, but should go into family-studies, not master

**Pull request closed without merging**